soroban-env-common/src/tuple.rs

```diff
@@ -57,6 +67,19 @@ macro_rules! impl_for_tuple {
             }
         }
 
+        impl<E: Env, $($typ),*> IntoVal<E, EnvVal<E, Object>> for ($($typ,)*)
+        where
+            $($typ: IntoVal<E, RawVal>),*
+        {
+            fn into_val(self, env: &E) -> EnvVal<E, Object> {
+                let rv: Object = self.into_val(env);
+                EnvVal{
+                    env: env.clone(),
+                    val: rv,
+                }
+            }
+        }
+
         impl<E: Env, $($typ),*> TryIntoVal<E, ($($typ,)*)> for RawVal
```

Comment:
Can you also add an implementation of this `for Object`? TryIntoVal for the RawVal type accompanies the TryFromVal at the top of the file (I guess these should be side-by-side).